doc/source/ref/wavelet-packets.rst

```diff
@@ -326,3 +373,76 @@ WaveletPacket2D and WaveletPacket2D tree Node2D
      If nodes at the given level are missing (i.e. the tree is partially
      decomposed) and the ``decompose`` is set to ``False``, only existing nodes
      will be returned.
+
+  .. method:: reconstruct([update=True])
+
+     Reconstruct data from the subnodes.
+
+     :param update: A boolean indicating whether the coefficients of the
+                    current node and its subnodes will be replaced with values
+                    from the reconstruction.
+
+WaveletPacketND and WaveletPacketND tree NodeND
```

Comment:
`WaveletPacketND tree NodeND` looks a little odd.

Reply:
I just changed it to `WaveletPacketND and NodeND` here (as well as for the 1D and 2D cases which used the same wording).